### DISCUSSION TITLE
log_buffer showing logs is not realtime.

### DISCUSSION THREAD

**Discussion by ChrisChou-freeman** [kkharji/xbase#133] [Q&A]
hi, first, this is great project. but i have some issue.
when i open "toggle_split_log_buffer" then i'm trying to use "print" function output some message, but it's not realtime, only when i quit application , can show before print message.
i just want know did i missing some configs or else. 

thank you.

**Comment by kkharji** [comment]
Hey @ChrisChou-freeman, I had faced exactly the same issue in the past #95 after that I never experienced the issue.

Hmm could you share more details about you setup and last few lines of logs /tmp/xbase.log after closing the simulator

**Reply by ChrisChou-freeman** [reply]
```
 INFO Runtime{name="Landmark"}: [Initializing] ------------------------
 INFO Broadcaster{name="Landmark"}: Created
 INFO Runtime{name="Landmark"}: targets: {"MacLandmarks": TargetInfo { platform: "macOS", configurations: ["Debug", "Release"] }, "MacLandmarksTests": TargetInfo { platform: "macOS", configurations: ["Debug", "Release"] }, "LandmarkUITests": TargetInfo { platform: "iOS", configurations: ["Debug", "Release"] }, "LandmarkTests": TargetInfo { platform: "iOS", configurations: ["Debug", "Release"] }, "Landmark": TargetInfo { platform: "iOS", configurations: ["Debug", "Release"] }, "MacLandmarksUITests": TargetInfo { platform: "macOS", configurations: ["Debug", "Release"] }}
 INFO Broadcaster{name="Landmark"}: Connected [46752]
 INFO Runtime{name="Landmark"}: Connected [46752]
 INFO FSWatcher{name="Landmark"}: Watching
 INFO Runtime{name="Landmark"}: [Initialized] -------------------------
 INFO FSWatcher{name="Landmark"}: Running MacLandmarks
 INFO FSWatcher{name="Landmark"}: Ran MacLandmarks
 INFO [MacLandmarks] Runner Closed
```
i try to run apple official SwiftUI Tutorials [link](https://developer.apple.com/tutorials/swiftui), macOS app.

i have other issue, when i try to go to definition, unable to choose which Targets to jump to, in this case, i have two Targets, one of is iOS Target another is macOS Target.

**Reply by ChrisChou-freeman** [reply]
i also try to run this app on iPhoneSE2 simulator, this is normal, i can see print message realtime. but when i first run iOS app on iPhoneSE2 simulator nothing happened, so i trying to run many times, nothing happened too. so i quit NeoVim and reopened, the simulator pop up a alert window, message is "Unable to boot device in current state: Booted",  then simulator try to run app many times before i tried. I want you to know just in case。
